nncf/tensorflow/sparsity/magnitude/algorithm.py

```diff
@@ -256,6 +256,31 @@ def compression_stage(self) -> CompressionStage:
             return CompressionStage.FULLY_COMPRESSED
         return CompressionStage.PARTIALLY_COMPRESSED
 
+    @property
+    def maximal_compression_rate(self) -> float:
```

Comment:
Could you explain why this implementation is consistent with "compression_rate"?

Reply:
Decided to return `maximal_compression_rate` of 1.0 for sparsity compression algorithm